src/components/ValueCard/ValueCard.story.jsx

```diff
@@ -72,6 +72,31 @@ storiesOf('Watson IoT|ValueCard', module)
       </div>
     );
   })
+
+  .add('small / wrapping', () => {
```

Comment:
There is a ton of stories already. Is there any way to combine this with another story?

Reply:
Ok, merged it into the "small / units" story that now shows to cards